processor/spanmetricsprocessor/README.md

```diff
@@ -54,6 +54,8 @@ The following settings can be optionally configured:
   If the `name`d attribute is missing in the span, the optional provided `default` is used.
   
   If no `default` is provided, this dimension will be **omitted** from the metric.
+- `metric_key_to_dimensions_cache_size`: the size of `metric_key_to_dimensions_cache`. If not provided, will use default
```

Comment:
minor nit: I think we should clarify here that size == number of items in cache just to be less ambiguous (since size can refer to a number of things e.g mb of memory, etc.)